access_log/access_log_record.go

```diff
@@ -94,3 +100,15 @@ func (r *AccessLogRecord) LogMessage() string {
 	recordBuffer := r.makeRecord()
 	return recordBuffer.String()
 }
+
+func (r *AccessLogRecord) ExtraHeaders() string {
+	extraHeadersToLog := make(map[string]string)
+	for _, header := range r.ExtraHeadersToLog {
+		extraHeadersToLog[header] = r.FormatRequestHeader(header)
+	}
+	marshalledExtraHeadersToLog, err := json.Marshal(extraHeadersToLog)
```

Comment:
Why are you using JSON here? The rest of the log line uses the `header_name: "Header value"` format, why not reusing the same format?